react-hot-button-component/src/index.jsx

```diff
@@ -0,0 +1 @@
+import './hot-button';
```

Comment:
Try only defining your component in `hot-button.jsx` and then exporting it. You can import the component, React, and ReactDOM into `index.jsx`.